client/webserver/site/src/html/dexsettings.tmpl

```diff
@@ -22,6 +22,9 @@
       <button id="updateCertBtn" class="bg2 selected">[[[Update TLS Certificate]]]</button>
```

Comment:
Not from this PR, but from https://github.com/decred/dcrdex/pull/1602#discussion_r878660443, I think only part of solution made it into the final product. 

Can you get rid of the `<h4>` up above and do something like

```HTML
<div class="flex-center fs16 mb-2">
  <span class="me-2 connection ico-connection d-hide" id="connectedIcon"></span>
  <span class="me-2 disconnected ico-disconnected d-hide" id="disconnectedIcon"></span>
  <span id="connectionStatus"></span>
</div>
```